modules/fixInternalDocs

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
I think we should stuff this down into modules/internal just to make it even more obvious it's for internal modules only